### PR TITLE
#164069205 Upgrading Author's Heaven from Django 1.1 to Django 2.1

### PR DESCRIPTION
**Pull request by BagzieGracious**
#### What does this PR do?
* This PR upgrades Author's Heaven from Django 1.1 to Django 2.1 and installs all the necessary packages for the app to work.
#### Description of Task to be completed?
* Replaced Django 1.1 with Django 2.1 in the requirements file
* Made migrations to sql database.
* Fixed JWTAuthentications backends bug.
* Fixed the url bug.

#### How should this be manually tested?
* Clone the repo and cd into Ah-backend-aquaman
* git checkout ch-upgrade-django-164069205
* pip install -r requirements.txt 
* python manage.py makemmigrations
* python manage.py migrate
* python manage.py runserver
#### Any background context you want to provide?
* N/A
#### What are the relevant pivotal tracker stories?
[#164069205](https://www.pivotaltracker.com/story/show/164069205)

### CODE REVIEW

File: authors/apps/authentication/backends.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+from django.conf import settings
```
Comment (marthamareal) [inline authors/apps/authentication/backends.py]: You can consider removing unused imports and also following PEP8 conventions, we leave a line at the end of a file.

**Comment by joelethan** [comment]
Good work @BagzieGracious

**Review by marthamareal — COMMENTED** [review]
I think we do not need the `db.sqlite3` file since we are using postgres

**Comment by CryceTruly** [comment]
LGTM

**Review by wasibani-roy — COMMENTED** [review]
Good work I have tested it and its working fine now

**Comment by Peace-Apple** [comment]
LGTM

**Comment by joelethan** [comment]
LGTM

**Review by joelethan — APPROVED** [review]

**Review by CryceTruly — APPROVED** [review]

**Review by wasibani-roy — APPROVED** [review]

**Review by Peace-Apple — COMMENTED** [review]
LGTM

**Pull request merged**